src/Microsoft.CodeQuality.Analyzers/Core/Maintainability/ReviewUnusedParameters.cs

```diff
@@ -206,7 +206,16 @@ public void OperationBlockEndAction(OperationBlockAnalysisContext context)
                 // Note that VB method bodies with 1 action have 3 operations.
                 // The first is the actual operation, the second is a label statement, and the third is a return
                 // statement. The last two are implicit in these scenarios.
-                if (context.OperationBlocks.Length == 1)
+
+                // Filter out Attribute operation blocks.
+                var operationBlocks = context.OperationBlocks;
+                if (operationBlocks.Any(operation => operation.IsAttribute()))
+                {
+                    operationBlocks = operationBlocks.Where(operation => !operation.IsAttribute()).ToImmutableArray();
```

Comment:
can block ever been OperationKind.None exception attributes? it looks like it can just filter out all OperationKind.None block?

Reply:
Attribute is the only case that I know of. If we find more cases, we will need to change this code to correctly decide whether to ignore or the entire block or operate on the Children.

Reply:
@mavasani curious. so, now in operation block, it put attributes as part of operation block like method body? rather than we calling attribute as its own operationblock?